gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```

Comment:
Is there a reason we are setting this now ?

Reply:
`gradlew` and `gradlew.bat` automatically generated after `$ gradle wrapper` command run ;). so i would expect this is OK